install_config/install/deploy_router.adoc

```diff
@@ -108,6 +108,31 @@ link:admin_guide/scheduler.html[scheduler configuration]. As an example, you can
 achieve this by dedicating infrastructure nodes to run services such as routers.
 ====
 
+[IMPORTANT]
+====
+It is recommended using separate distinct *openshift-router* credentials
+with your router. The credentials can be provided using the `--credentials`
+flag to the `oadm router` command. Alternatively, the default admin
+credentials can be used from the `$KUBECONFIG` environment variable.
+
+ifdef::openshift-enterprise[]
+----
+$ oadm router --dry-run --service-account=router \
+    --credentials='/etc/origin/master/openshift-router.kubeconfig' //<1>
+----
+endif::[]
+ifdef::openshift-origin[]
+----
+$ oadm router --dry-run --service-account=router \
+    --credentials=${ROUTER_KUBECONFIG:-"$KUBECONFIG"} //<1>
+----
+endif::[]
+<1> `--credentials` is the path to the
+link:../../cli_reference/manage_cli_profiles.html[CLI configuration file]
+for the *openshift-router*. It is recommended using an *openshift-router*
+specific profile with appropriate permissions.
+====
+
```

Comment:
@ramr It might be a little over the top, but what if you extended the diffs so that there was a paragraph explaining the Enterprise side with an example in the ent diff, then the same with the Origin side of things? I don't think it'll be as neat as what you have above, but it might be more correct.